src/components/TabPanels/SbTabPanels.vue

```diff
@@ -0,0 +1,36 @@
+<script>
```

Comment:
@jameslovallo you could change to SFC instead of using template functions, I say this because we are moving all components that use render functions to SFC files, if you can do that, your PR will be merged.

Thanks in advance ❤️